Realm/ObjectStore/results.cpp

```diff
@@ -146,6 +168,11 @@ void Results::update_tableview()
             m_mode = Mode::TableView;
             break;
         case Mode::TableView:
+            if (!m_background_query && m_realm->can_deliver_notifications() &&  !m_realm->is_in_transaction()) {
```

Comment:
Move the `!m_realm->is_in_transaction()` check up before `m_realm->can_deliver_notifications()` since it's much cheaper? Also, inconsistent spacing around `&&`.